plugins/search/src/components/SearchBar/SearchBar.tsx

```diff
@@ -14,14 +14,61 @@
  * limitations under the License.
  */
 
-import React, { ChangeEvent, useEffect, useState } from 'react';
+import React, { ChangeEvent, useEffect, KeyboardEvent, useState } from 'react';
 import { useDebounce } from 'react-use';
 import { InputBase, InputAdornment, IconButton } from '@material-ui/core';
 import SearchIcon from '@material-ui/icons/Search';
 import ClearButton from '@material-ui/icons/Clear';
 
 import { useSearch } from '../SearchContext';
 
+type PresenterProps = {
+  value: string;
+  onChange: (e: ChangeEvent<HTMLInputElement>) => void;
+  handleClear?: () => void;
+  onKeyDown?: (e: KeyboardEvent<HTMLInputElement>) => void;
+  className?: string;
+  placeholder?: string;
+};
+
+export const SearchBarPresenter = ({
```

Comment:
Maybe `SearchBarBase` to match Material's scheme?  🤷‍♂️

Reply:
Yea that makes a lot of sense. Good call!

Fixed in 56a15e.